spec/helpers/application_helper_spec.rb

```diff
@@ -76,37 +76,45 @@
       end
 
       context "when the domain is 'https://training'" do
-        it "returns 'training" do
+        it "returns 'training'" do
           ClimateControl.modify DOMAIN: "https://training.report-official-development-assistance.service.gov.uk" do
             expect(helper.environment_name).to eql("training")
           end
         end
       end
 
       context "when the domain is 'https://pentest'" do
-        it "returns 'training" do
+        it "returns 'training'" do
```

Comment:
Are there supposed to be two 'training's here? I thought pentest was always training

Reply:
I think this was defensive development - when we introduced the banners, there was a pentest domain that was used for training, and we wanted to display training, but I was anticipating that the domain might one day change to actually say training, and we'd still want it to be recognised.

If we're sure that there no longer exists a pentest domain, we could remove that branch and simplify a bunch of code.

Reply:
I'm not in a huge rush, and I'm keen to not accumulate tech debt that we already see coming, so I'll check that the pentest domain is not used anymore!

Reply:
@shuldt I've confirmed that the pentest domain is not in use anymore, and updated the code accordingly.